protected/humhub/modules/space/models/Space.php

```diff
@@ -119,7 +119,7 @@ public function rules()
             [['visibility'], 'in', 'range' => [0, 1, 2]],
             [['visibility'], 'checkVisibility'],
             [['url'], 'unique', 'skipOnEmpty' => 'true'],
-            [['guid', 'name', 'url'], 'string', 'max' => 45, 'min' => 2],
+            [['guid', 'name', 'url'], 'string', 'max' => Yii::$app->getModule('space')->maximumSpaceUrlLength, 'min' => Yii::$app->getModule('space')->minimumSpaceUrlLength],
```

Comment:
MaximumSpaceUrlLength and minimumSpaceUrlLength are applied for guid, name and url. Could you move url to a seperate rule and leave guid and name?